src/Service/ProgrammesAggregationService.php

```diff
@@ -42,6 +42,24 @@ public function findStreamableDescendantClips(
         return $this->findStreamableDescendantsByType($programme, 'Clip', $limit, $page, $ttl, $nullTtl);
     }
 
+    public function countStreamableDescendantClips(
+        Programme $programme,
+        $ttl = CacheInterface::NORMAL,
+        $nullTtl = CacheInterface::NORMAL
+    ) : int {
+        $key = $this->cache->keyHelper(__CLASS__, __FUNCTION__, $programme->getPid(), 'ClipsCount', null, null, $ttl);
```

Comment:
You don't need the two null parameters here, or that clipscount string

Reply:
Sorry about the 2 nulls, but I tought the ClipsCount was a sort of prefix to make the cache key unique?

Reply:
ok I understand now, it's the __FUNCTION__ that will help generate it, got it!